dockers/skylab/loom-output/create_loom_optimus.py

```diff
@@ -103,8 +103,8 @@ def  generate_row_attr(args):
 
     nrows = len(gene_ids)
 
-    #Generate row attribute dictionary
-    row_attrs = { "gene_names":gene_names, "ensembl_ids": gene_ids}
+    # Generate row attribute dictionary
+    row_attrs = { "gene_names":gene_names, "ensembl_ids": gene_ids, "Gene": gene_names}
```

Comment:
why are we duplicating the gene_names information?

Reply:
To be compatible with Seurat and Scanpy